Aptoide/aptoide-client-v8
# [APP-1319] review tc and pp

**What does this PR do?**

This PR aims at reverting the previously implemented solution for gdpr. Before we used to have a setting when the user attempted to login. Then we changed that for a full screen dialog that the user had to accept otherwise it could not use the app. Now in this PR we are reverting to the 1st solution.

Also used this PR to fix the error message for the rakam install event error (cancel) message to be regarding an install instead of a download.

**Database changed?**

 No

**Where should the reviewer start?**


**How should this be manually tested?**

Open the app (clear session) check that the mentioned dialog does not show up anymore. Then also test the login flow and check that you have the checkbox and you can only login if you accept the checkbox.
**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-1319](https://aptoide.atlassian.net/browse/APP-1319)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
